offline-runtime/data/src/com/intellij/rt/coverage/offline/RawHitsReport.java

```diff
@@ -107,12 +88,4 @@ public static boolean isRawHitsFile(File file) throws IOException {
       CoverageIOUtil.close(is);
     }
   }
-
-  public static void dumpOnExit(final File file, final RawProjectData data) {
```

Comment:
I'll leave that for test needs

Reply:
Maybe it's better to put class `kotlinx.kover.offline.runtime.KoverInit` for tests?

Reply:
I think it is pretty much OK with a Vm option, while it does not depend on extra classes. And when kover calls `dumpIcReport`?

Reply:
onExit (property-dependent) or if API called manually